support-redemptiondb/cfn.yaml

```diff
@@ -0,0 +1,58 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  Stage:
+    Type: String
+    Description: Stage for the tables
+    AllowedValues:
+      - CODE
+      - PROD
+Resources:
+  RedemptionCodeDynamoTableDEV:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: "redemptionCode"
+          AttributeType: "S"
+      BillingMode: "PAY_PER_REQUEST"
+      KeySchema:
+        - AttributeName: "redemptionCode"
+          KeyType: "HASH"
+      TableName: !Sub redemption-codes-DEV-${Stage}
```

Comment:
I thought the first touch point stage is DEV but looking at https://github.com/guardian/support-frontend/blob/968f37325dc68e7b51349a9797633618024ed4a8/support-models/src/main/scala/com/gu/support/config/TouchPointEnvironments.scala#L13
perhaps it should be SANDBOX?

Reply:
looking at membership-common, members-data-api it is always referred to as DEV/UAT/PROD, so I think support is the outlier there
https://github.com/search?q=org%3Aguardian+sandbox+uat&type=Code

Reply:
Yes I'd like to change it to DEV/UAT/PROD at some point